### ISSUE TITLE
fatty acid omega-hydroxylase activity

### ISSUE THREAD

**Issue by nnouspikel** [geneontology/go-ontology#19899]
Dear GOA curators,

Could you please review the definition of  GO:0102033 fatty acid omega-hydroxylase activity?

We can replace the current one as below:
Definition: Catalysis of the reaction: an omega-methyl fatty acid + O2 + reduced [NADPH--hemoprotein reductase] = an omega-hydroxy fatty acid + H(+) + H2O + oxidized [NADPH--hemoprotein reductase] [RHEA:39023]

and create new_GO: long-chain fatty acid omega-hydroxylase activity
Definition: Catalysis of the reaction: an omega-methyl-long-chain fatty acid + O2 + reduced [NADPH--hemoprotein reductase] = an omega-hydroxy-long-chain fatty acid + H(+) + H2O + oxidized [NADPH--hemoprotein reductase] [Rhea:56748]
EC=1.14.14.80
as child of GO:0102033

Thank you very much for your help.

Nevila Hyka-Nouspikel

**Comment by krchristie** [comment]
Hi @nnouspikel 


> Could you please review the definition of GO:0102033 fatty acid omega-hydroxylase activity?
> 
> We can replace the current one as below:
> Definition: Catalysis of the reaction: an omega-methyl fatty acid + O2 + reduced [NADPH--hemoprotein reductase] = an omega-hydroxy fatty acid + H(+) + H2O + oxidized [NADPH--hemoprotein reductase] [RHEA:39023]
> 
> and create new_GO: long-chain fatty acid omega-hydroxylase activity
> Definition: Catalysis of the reaction: an omega-methyl-long-chain fatty acid + O2 + reduced [NADPH--hemoprotein reductase] = an omega-hydroxy-long-chain fatty acid + H(+) + H2O + oxidized [NADPH--hemoprotein reductase] [Rhea:56748]
> EC=1.14.14.80
> as child of GO:0102033

Thanks very much for this suggestion. I definitely agree that we should have two terms here. However, due to GO's rules that the ID goes with the definition (which can include the xrefs associated with a term as well as the definition field), I will change the term name of GO:0102033 to be the long chain term as that matches these aspects of this terms definition: 
- the existing definition specifies "long-chain fatty acid" (I will update the definition to use the Rhea definition as you suggest)
- Rhea ID
- EC #, noting that EC:1.14.14.80 which you suggest was previously EC:1.14.13.205 which is the EC # currently associated with GO:0102033

I will create a new term for "fatty acid omega-hydroxylase activity" with the definition and Rhea reference you suggest.

----
I also checked the AmiGO page and do not see any direct annotations to this term, info below. I also checked QuickGO and don't think there were any direct annotations there either.

AmiGO - http://amigo.geneontology.org/amigo/term/GO:0102033

Accession: GO:0102033
Name: fatty acid omega-hydroxylase activity
Ontology: molecular_function
Synonyms: cytochrome P450 fatty acid omega-hydroxylase activity
Alternate IDs: None
Definition: Catalysis of the reaction: long-chain fatty acid + NADPH + H+ + O2 = an omega-hydroxy-long-chain fatty acid + NADP+ + H2O. Source: PMID:18544608, GOC:pz 
Comment: None
History: See term history for GO:0102033 at QuickGO
Subset: None

Total annotations: 0
